src/map/skill.cpp

```diff
@@ -7568,8 +7568,6 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 					skill_delunit(su);
 				clif_skill_nodamage(src, bl, skill_id, skill_lv, 1);
 			}
-			else
-				clif_skill_fail(sd,skill_id,USESKILL_FAIL_LEVEL,0);
```

Comment:
Are you sure that this is not producing an error message?
For example if you have a full inventory?

Reply:
Yeah, I'm sure. And you can't use skills over 90% of weight, so that could never happen.

Reply:
I'm not talking about 90% weight limit, but 100 item slots.

Reply:
![imagen](https://user-images.githubusercontent.com/8776528/35631204-96b57a6e-06a3-11e8-87d6-547e4baacd72.png)

Reply:
Is the holy water dropped on the floor or not?

Reply:
Nope, is not generated.